src/configure-nilrt-snac

```diff
@@ -172,6 +172,17 @@ remove_niauth() {
 	trap - EXIT
 }
 
+enable_pwquality() {
+	log INFO Enabling password quality checks...
+	log INFO Enabling password history...
+	sed -i 's/^\(password.*pam_unix.so.*\)$/\1 remember=5/' /etc/pam.d/common-password
+	log INFO Enabling password complexity using libpwquality...
+	cat <<-EOF >>/etc/pam.d/common-password
```

Comment:
Some configuration steps above occur only if one doesn't exist. Do we care what the outcome is if this script is run twice?

Reply:
AB#2838202 will take care of this concern as the script is converted to Python